### ISSUE TITLE
Key Error: Cipher, Key Error url

### ISSUE THREAD

**Issue by pieeee** [pytube/pytube#648]
```Traceback (most recent call last):
  File "C:\Users\Ahnaf\Desktop\Ydn\venv\lib\site-packages\pytube\extract.py", line 288, in apply_descrambler
    stream_data[key] = [
  File "C:\Users\Ahnaf\Desktop\Ydn\venv\lib\site-packages\pytube\extract.py", line 290, in <listcomp>
    "url": format_item["url"],
KeyError: 'url'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:/Users/Ahnaf/Desktop/Ydn/test2.py", line 3, in <module>
    yt = YouTube('https://www.youtube.com/watch?v=TME8EQrifgc&list=RDTME8EQrifgc&start_radio=1')
  File "C:\Users\Ahnaf\Desktop\Ydn\venv\lib\site-packages\pytube\__main__.py", line 92, in __init__
    self.descramble()
  File "C:\Users\Ahnaf\Desktop\Ydn\venv\lib\site-packages\pytube\__main__.py", line 132, in descramble
    apply_descrambler(self.player_config_args, fmt)
  File "C:\Users\Ahnaf\Desktop\Ydn\venv\lib\site-packages\pytube\extract.py", line 300, in apply_descrambler
    cipher_url = [
  File "C:\Users\Ahnaf\Desktop\Ydn\venv\lib\site-packages\pytube\extract.py", line 301, in <listcomp>
    parse_qs(formats[i]["cipher"]) for i, data in enumerate(formats)
KeyError: 'cipher'```

**Comment by BunnyBrewery** [comment]
#642 

It is said that the fix metioned in #642 resolves your issue. 

I have the same issue but the fix didn't work for me.

**Comment by choonsiong** [comment]
It works for me after changing following line inside the file
`/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pytube/extract.py`

Around line 301, inside the `except KeyError:`:
`parse_qs(formats[i]["signatureCipher"]) for i, data in enumerate(formats)`
